### DISCUSSION TITLE
Apple Watch or Google Wear

### DISCUSSION THREAD

**Discussion by yoyomo** [tauri-apps/tauri#10142] [Ideas]
With the new addition of mobile support, would it be crazy to think about supporting both Apple Watch OS and Google Wear OS ? Noted it has a very limited view and limited functionality, but would be very cool to have it be managed in Tauri as well.

**Comment by FabianLars** [comment]
For now it's a bit crazy, yes. Neither WatchOS nor WearOS expose the webview (though it's possible to hack it <https://x.com/jonasKruckie/status/1668234348871516161> but of course, you can't distribute that). I honestly doubt we will ever see it, considering how resource hungry browser engines are (at least in the context of watches).

Therefore this would fall in a similar-ish category to Widgets on phones that also don't have webview support but people still ask for support in Tauri which we didn't rule out yet (to be fair though, widgets are wayyy more likely to happen in Tauri than supporting a whole new platform).

**Reply by yoyomo** [reply]
Interesting. Thank you for responding! I did not think of widgets or watches not supporting webview, which makes complete sense why it wouldn't be supported

**Reply by JonasKruckenberg** [reply]
The only thing I can add to this is that during my chat with apple during last years WWDC they didn't seem to be totally against opening up WKWebView on WatchOS since it already technically exists for mail and the oauth login flow. 
That really doesn't mean much as-is, but if community need for this turns out to be large enough maybe apple could be lobbied into supporting it. 

This doesn't mean though that its a good idea. We have always been quite open about where Tauri shines and where it doesn't and resource constrained devices like watches are not a good fit for web tech in 2024.
